python/hsfs/feature_view.py

```diff
@@ -710,6 +1193,41 @@ def training_data(
         """
         Get training data from feature groups.
 
+        !!! example "Create random splits"
+            ```python
+            # get feature store instance 
+            fs = ...
+
+            # get feature view instance 
+            feature_view = fs.get_feature_view(...)
+
+            # get training data
+            feature_df, label_df = feature_view.training_data(
+                description='Descriprion of a dataset',
+            )
+            ```
+
+        !!! example "Create time-series based splits"
+            ```python
+            # get feature store instance 
+            fs = ...
+
+            # get feature view instance 
+            feature_view = fs.get_feature_view(...)
+
+            # set up a date
+            start_time = "2022-05-01 00:00:00"
+            end_time = "2022-06-04 23:59:59"
+            # you can also pass dates as datetime objects
+            
+            # get training data
+            feature_df, label_df = feature_view.training_data(
+                start_time=start_time,
+                end_time=end_time,    
+                description='Descriprion of a dataset'
+            )
+            ```
+
         !!! info
```

Comment:
This info box is duplicated. Please remove it - it's already in the above section on recreating training datasets.